.vscode/launch.json

```diff
@@ -10,6 +10,7 @@
             "request": "launch",
             "module": "quart",
             "cwd": "${workspaceFolder}/app/backend",
+            "python": "${workspaceFolder}/app/backend/backend_env/bin/python",
```

Comment:
Is backend_env the name we use in a setup script?

Reply:
yes, in https://github.com/Azure-Samples/azure-search-openai-demo/blob/a1170562395ad122d2be0e8e29d33cac7ff169c2/app/start.sh#L20
is there a less hard codey approach?

Reply:
If you leave it out, VS Code will select the configured interpreter or there is a special variable to inherit that. https://code.visualstudio.com/docs/python/debugging#_python

Reply:
Ah I think I don't typically configure an interpreter for my workspace, so I assume it defaults to the Python of the dev container. That won't always work since people may not have also installed into the dev container directly. So I think we need it to be backend_env specifically, since we're assuming people are setting that up? Or, should there be a prerequisite step to the debugger that runs pip install at the root?

Reply:
Ping on this discussion. Just working on my slides about local development and remembered this was pending.